docusaurus.config.js

```diff
@@ -62,12 +58,13 @@ const config = {
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
       // Replace with your project's social card
-      image: 'img/docusaurus-social-card.jpg',
+      image: 'img/logo-emblem.svg',
```

Comment:
AFAICT a social card defines what shows up in the preview when sharing in various social media. I was trying to test this out with slack but I can't seem to get anything to show up. Guess slack doesn't count as social media (fair)

Anyways I don't feel like tweeting (or "X"ing 🙄 ) to find out, but again, it seems like a good idea to have this set to our logo and not some random docusaurus thing